modules/swiss/src/main/SwissApi.scala

```diff
@@ -355,6 +355,22 @@ final class SwissApi(
       }
     }
 
+  def playerToPairingGames(playerView: SwissPlayer.ViewExt): Fu[List[SwissPairingGames]] =
+    playerView.pairings.values.toSeq.map(_.pairing).map { p =>
```

Comment:
if you do a .toList instead of .toSeq, then you wont need the .toList on line 369?